packages/mwp-config/stylelint/stylelint-config-default.js

```diff
@@ -0,0 +1,206 @@
+		'function-url-scheme-blacklist': ['/^data:image/'],
+		'function-whitespace-after': 'always',
+		indentation: 'tab',
+		'length-zero-no-unit': true,
+		'max-nesting-depth': 3,
+		'media-feature-colon-space-after': 'always',
+		'media-feature-colon-space-before': 'never',
+		'media-feature-name-case': 'lower',
+		'media-feature-name-no-unknown': true,
+		'media-feature-parentheses-space-inside': 'never',
+		'media-feature-range-operator-space-after': 'always',
+		'media-feature-range-operator-space-before': 'always',
+		'media-query-list-comma-space-after': 'always',
+		'media-query-list-comma-space-before': 'never',
+		'no-duplicate-at-import-rules': true,
+		'no-duplicate-selectors': true,
+		'no-empty-source': true,
+		'no-eol-whitespace': true,
+		'no-extra-semicolons': true,
+		'no-unknown-animations': true,
+		'number-leading-zero': 'always',
+		'number-max-precision': 2,
+		'number-no-trailing-zeros': true,
+		'property-blacklist': 'text-rendering',
+		'property-case': 'lower',
+		'property-no-unknown': true,
+		'property-no-vendor-prefix': true,
+		'rule-empty-line-before': 'always',
+		'selector-attribute-brackets-space-inside': 'never',
+		'selector-attribute-operator-blacklist': '|=',
+		'selector-attribute-operator-space-after': 'never',
+		'selector-attribute-operator-space-before': 'never',
+		'selector-attribute-quotes': 'always',
+		'selector-class-pattern': [
+			'^(?:[a-z]+[a-zA-Z0-9]+_)*(?:[a-z]+[a-zA-Z0-9]+)*(?:-[a-zA-Z0-9]+)*(?:--[a-zA-Z0-9]+)?$',
+			{ resolveNestedSelectors: true },
+		],
+		'selector-combinator-space-after': 'always',
+		'selector-combinator-space-before': 'always',
+		'selector-descendant-combinator-no-non-space': true,
+		'selector-list-comma-newline-after': 'always',
+		'selector-list-comma-newline-before': 'never-multi-line',
+		'selector-list-comma-space-after': 'never',
+		'selector-list-comma-space-before': 'never',
+		'selector-max-attribute': 1,
+		'selector-max-class': 1,
+		'selector-max-combinators': 1,
+		'selector-max-compound-selectors': 2,
+		'selector-max-empty-lines': 0,
+		'selector-max-id': 0,
+		'selector-max-specificity': '0,2,1',
+		'selector-max-universal': 0,
+		'selector-no-qualifying-type': true,
+		'selector-no-vendor-prefix': true,
+		'selector-pseudo-class-case': 'lower',
+		'selector-pseudo-class-no-unknown': true,
+		'selector-pseudo-class-parentheses-space-inside': 'never',
+		'selector-pseudo-element-case': 'lower',
+		'selector-pseudo-element-colon-notation': 'double',
+		'selector-pseudo-element-no-unknown': true,
+		'selector-type-case': 'lower',
+		'selector-type-no-unknown': true,
+		'shorthand-property-no-redundant-values': true,
+		'string-no-newline': true,
+		'string-quotes': 'double',
+		'time-min-milliseconds': 100,
+		'unit-case': 'lower',
+		'unit-no-unknown': true,
+		'value-keyword-case': 'lower',
+		'value-list-max-empty-lines': 0,
+		'value-no-vendor-prefix': true,
+
+		// the following rules are intentionally disabled
+		'at-rule-blacklist': null,
```

Comment:
dumb question - do the rules included in `stylelint` or `stylelint-scss` default to `null`? If so, we may not need to explicitly declare them here.

Reply:
They default to null, but I kind of like being explicit about what we've disabled. Happy to remove if you prefer not having them :)

Reply:
The reason I can think of to keep these around is to document their absence as being intentional, but that doesn't provide much context around _why_ we intend not to use these.

I'd vote to remove them to fall back on default `null` values for rules we don't want.